### DISCUSSION TITLE
Ability to change the key positions for mania in touch devices

### DISCUSSION THREAD

**Discussion by roxandtol** [ppy/osu#19364] [Ideas] [answered]
in the mania ruleset, if you're playing with a touch device (your phone for example), when you try to click 2 keys that are next to each other, it might be a bit hard to press them, so a solution would be to either move the part where you click a bit, have an option to scale the size of the the part you click, or be able to independently move each part where you click

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/16783
